components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/extension/EvalPermissionTreeFunction.java

```diff
@@ -79,7 +77,7 @@ public EvaluationResult evaluate(List<Evaluatable> inputs, EvaluationCtx context
                     isAuthorised = EntitlementServiceComponent.getRealmservice().getBootstrapRealm().
                             getAuthorizationManager().isUserAuthorized(subject, resource, "ui.execute");
                 } catch (UserStoreException e) {
-                    log.error("error while user authorization", e);
+                    log.error("Error while user authorization.", e);
```

Comment:
Can we add some context information?
eg: subject, resource etc.
Will make it easier to figure out in case of an error